src/Illuminate/Foundation/Testing/Concerns/MocksApplicationServices.php

```diff
@@ -88,7 +89,14 @@ protected function withoutEvents()
             $this->firedEvents[] = $called;
         });
 
+        $mock->shouldReceive('until')->andReturnUsing(function ($called) {
+            $this->firedEvents[] = $called;
+
+            return true;
```

Comment:
We don't have a way of testing the `false` condition of model events here. Anyone have any ideas?